src/plugins/orangehrmPerformancePlugin/Dto/ReviewEmployeeSupervisorSearchFilterParams.php

```diff
@@ -23,6 +23,8 @@
 
 class ReviewEmployeeSupervisorSearchFilterParams extends FilterParams
 {
+    public const ALLOWED_SORT_FIELDS = ['employee.empNumber'];
```

Comment:
seems some validation rules missing in `ReviewEmployeeSupervisorAPI::getValidationRuleForGetAll` 
```php
        return new ParamRuleCollection(
            $this->getValidationDecorator()->notRequiredParamRule(
                new ParamRule(
                    CommonParams::PARAMETER_EMP_NUMBER,
                    new Rule(Rules::POSITIVE)
                ),
            ),
            $this->getValidationDecorator()->notRequiredParamRule(
                new ParamRule(
                    self::FILTER_NAME_OR_ID,
                    new Rule(Rules::STRING_TYPE),
                    new Rule(
                        Rules::LENGTH,
                        [null, self::PARAM_RULE_FILTER_NAME_OR_ID_MAX_LENGTH]
                    ),
                )
            ),
            // following line seems missing
            ...$this->getSortingAndPaginationParamsRules(ReviewEmployeeSupervisorSearchFilterParams::ALLOWED_SORT_FIELDS),
        );
```